sync/etcdv3/etcd.go

```diff
@@ -195,7 +215,7 @@ func (s *Sync) Lock(path string, block bool) (chan struct{}, error) {
 				return nil, errors.New(msg)
 			}
 			time.Sleep(masterTTL * time.Second)
```

Comment:
this sleep does not belong here. please remove the "block" parameter and move the sleep to the caller site